src/it/add-third-party-missing-file/prebuild.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * #%L
+ * License Maven Plugin
+ * %%
+ * Copyright (C) 2019, Falco Nikolas
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.jar.JarOutputStream;
+import java.util.jar.Manifest;
+import java.util.zip.ZipEntry;
+
+JarOutputStream licenseRepo = new JarOutputStream(new FileOutputStream(new File(basedir, "license-repo.jar")));
+
+licenseRepo.putNextEntry(new ZipEntry("missing-license.properties"));
+licenseRepo.write("org.json--json--20070829=The JSON License".getBytes());
```

Comment:
> new File(basedir, "license-repo.jar")

I think `basedir + "/target"` would be a better place for the file. Otherwise someone someday is going to add it to git.

Reply:
the license-repo.jar file is created already in target/it/add-third-party-missing-file, do you want I move it to target/it/add-third-party-missing-file/target?